src/KafkaFlow/Middlewares/Serializer/Configuration/ConsumerMiddlewareConfigurationBuilder.cs

```diff
@@ -1,51 +1,52 @@
-﻿namespace KafkaFlow
+﻿namespace KafkaFlow.Middlewares.Serializer.Configuration
```

Comment:
Given that this file only has configuration methods utilized during setup, it's appropriate to adopt the root namespace `KafkaFlow`.

Reply:
Makes sense 👍 
Done